package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react_todo-app-add-and-delete",
-  "homepage": "react_todo-app-add-and-delete",
+  "name": "react_todo-app-loading-todos",
+  "homepage": "react_todo-app-loading-todos",
```

Comment:
demo link still not works.
Looks like problem is here.
Why you changed homepage? 
![Screenshot 2025-01-16 at 15 29 47](https://github.com/user-attachments/assets/d42e04c9-a694-4130-b248-cb97d8b40016)

please check demo link before next re-request and feel free to ask in chat if will have any problems with fixing ...